code/processes/wdb.q

```diff
@@ -222,8 +228,11 @@ flushend:{
 	 .wdb.reloadcomplete:1b];
 	};
 
-/- initialise d
-d:()!()
+/- initialise d, keyed tale to track status of local reloads
+d:([`int$handle:()]`symbol$process:();`boolean$status:();`symbol$result:());
```

Comment:
Can you come up with a better name for this table
I know this is what it was previously called, and it may well have been me that called it that, but it's awful

Reply:
Like "reloadstatus" or something...

Reply:
Have renamed to .wdb.reloadstatus